src/jit/assertionprop.cpp

```diff
@@ -68,8 +65,10 @@ void Compiler::optAddCopies()
         return;
     }
 
-    for (lclNum = 0, varDsc = lvaTable; lclNum < lvaCount; lclNum++, varDsc++)
+    for (unsigned lclNum = 0; lclNum < lvaCount; lclNum++)
     {
+        LclVarDsc* varDsc = &lvaTable[lclNum];
```

Comment:
To be clear - it's only preferable because so much code already uses pointers and converting that code to use references may not be an option. Otherwise I see no reason to use `LclVarDsc*` instead of `LclVarDsc&` anywhere where nulls aren't needed.